src/map/script-fun.cpp

```diff
@@ -3117,6 +3117,32 @@ BuiltinFunction builtin_functions[] =
     BUILTIN(getmap, ""_s, 's'),
     BUILTIN(mapexit, ""_s, '\0'),
     BUILTIN(freeloop, "i"_s, '\0'),
+    // Magic builtins needed
+    //BUILTIN(sfx, ".ii"_s),
+    //BUILTIN(instaheal, "eii"_s),
```

Comment:
instaheal is `heal XX, YY`, itemheal is `heal XX, YY, 1`
no need for new builtins